externals/kyuubi-spark-sql-engine/src/main/scala/org/apache/kyuubi/engine/spark/session/SparkSQLSessionManager.scala

```diff
@@ -99,10 +99,11 @@ class SparkSQLSessionManager private (name: String, spark: SparkSession)
   }
 
   private def setModifiableConfig(spark: SparkSession, key: String, value: String): Unit = {
-    if (spark.conf.isModifiable(key)) {
+    try {
       spark.conf.set(key, value)
-    } else {
-      warn(s"Spark config $key is static and will be ignored")
+    } catch {
+      case _: AnalysisException =>
+        warn(s"Spark config $key is not modifiable during running, ignored it")
```

Comment:
nit: the original message of `AnalysisException` sounds more reasonable, the word `running` looks confusing